test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -672,6 +672,58 @@ describe('Dropdown', () => {
         .at(options.length - 1)
         .should.have.prop('selected', true)
     })
+    it('does not move up on arrow up when first item is selected when open when \'wrapSelection\' is on', () => {
```

Comment:
nit: Double quotes when string contains single quotes (prevents escaping)

Side note, we really need to move to `prettier` :)